docs/_docs/permalinks.md

```diff
@@ -153,10 +181,11 @@ permalink is defined according to the format `/:categories/:year/:month/:day/:ti
 </table>
 </div>
 
-## Built-in permalink styles
+Note that all template variables relating to time or categories are available to posts only.
+
+## Built-in permalink styles {#builtinpermalinkstyles}
 
-While you can specify a custom permalink style using [template variables](#template-variables),
-Jekyll also provides the following built-in styles for convenience.
+Although you can specify a custom permalink pattern using template variables, Jekyll also provides the following built-in styles for convenience.
```

Comment:
updated the link. left "built-in" instead of "predefined" since "built-in" is used elsewhere. both terms seem good to me.